starboard/build/config/BUILD.gn

```diff
@@ -241,20 +240,17 @@ config("default_cpp_standard") {
   if (current_toolchain == default_toolchain) {
     if (sb_is_modular || is_android) {
       cflags_cc = [ "-std=c++17" ]
-    }
-
-    # These flags apply to non-modular windows platforms which includes win32, xb1.
-    if (is_host_win && !sb_is_modular) {
-      cflags = [ "/std:c++17" ]
-    }
-
-    if (is_linux && !sb_is_modular) {
+    } else if (is_host_win && !sb_is_modular) {
```

Comment:
You can simplify a lot of these conditionals now that they're `else if`s, i.e. remove !sb_is_modular

Reply:
done